package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@skyux/sample",
+  "name": "@skyux/phonefield",
```

Comment:
Should this be `phone-field` ?

Reply:
Ya. Those were holdovers of the original repo naming. I've fixed all instances now.